src/thor/costmatrix.cc

```diff
@@ -132,19 +132,17 @@ void CostMatrix::SourceToTarget(Api& request,
                                 baldr::GraphReader& graphreader,
                                 const sif::mode_costing_t& mode_costing,
                                 const sif::travel_mode_t mode,
-                                const float max_matrix_distance,
-                                const bool has_time,
-                                const bool invariant,
-                                const ShapeFormat& shape_format) {
```

Comment:
I pulled all these out of the signature and set them inside the function instead. was pointless to have them in the signature, it'll all in the `request`, except for `has_time` which is being set via the base class's `set_has_time()`